### PR TITLE
Feat/account page

### PR DESCRIPTION
**Pull request by ezeaniiandrew**
Updated the UI:
- Added card menu and used the appropriate icon for the cards

### CODE REVIEW

**Review by Hemazyn — APPROVED** [review]
Good job.

File: src/pages/Browse/index.jsx [hunk 1]
```diff
@@ -10,7 +10,13 @@ const Browse = () => {
       {/* <Header /> */}
       <div className={styles.wrapper}>
         {events.map((event) => (
-          <EventCard key={event} event={event} />
+          <EventCard key={event}>
+                    <EventCardIcon onClick={() => setIsBookmarked(!isBookmarked)}>
```
Comment (Leeyah-123) [inline src/pages/Browse/index.jsx]: Move all this into the event card itself.

**Review by Leeyah-123 — APPROVED** [review]

**Review by Leeyah-123 — CHANGES_REQUESTED** [review]
The filter dropdown is not positioned well

**Review by Leeyah-123 — CHANGES_REQUESTED** [review]
Also, remove you're not passing the icon into the event card somewhere.
I think on the Browse page.

image

**Review by Leeyah-123 — CHANGES_REQUESTED** [review]
These should be icons inside of buttons.
image

Also, the dropdowns are not accessible at all. Read up on accessible dropdowns and implement them. Both the event card dropdown and the filter dropdown.

**Comment by ezeaniiandrew** [comment]
> These should be icons inside of buttons. image
> 
> Also, the dropdowns are not accessible at all. Read up on accessible dropdowns and implement them. Both the event card dropdown and the filter dropdown.

I'll push a commit to fix the icon issue. Regarding accessibility, given the time constraints, I think it'll be better to implement that when we're done with the project so as not to delay the implementation of the important pages.

**Review by Leeyah-123 — APPROVED** [review]

**Pull request merged**